raster-cli/public/PbmImage.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "Image.hpp"
+
+#include <boost/optional.hpp>
+
+#include <fstream>
+
+class Operation;
+
+class PbmImage : public Image {
```

Comment:
NetPBM? PBM is a specific type of NetPBM image.

Reply:
I didn't notice that. Thanks!